README.md

```diff
@@ -48,6 +48,8 @@ environment created in step 3 is active.
 
 ```
 git clone https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME
+cd $CUSPATIAL_HOME
```

Comment:
Why not just `git clone --recurse-submodules https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME`

?

Reply:
Agree, `--recurse-submodules` is used in all the cudf scripts as well.

Reply:
Ok.